### PR TITLE
docs: update relate doc about the change in verificationCertStores

### PR DESCRIPTION
**Pull request by junczhu**
update the related config and description about the change of `verificationCertStores ` in [PR ](https://github.com/ratify-project/ratify/pull/1538)

### CODE REVIEW

**Review by akashsinghal — APPROVED** [review]
LGTM

File: docs/plugins/verifier/notation.md [hunk 1]
```diff
@@ -85,6 +75,23 @@ spec:
...
+Note: CLI is NOT SUPPORTED.
+
+
+> NOTE 1: `verificationCertStore` is able to reference a [KeyManagementProvider](../../reference/custom%20resources/key-management-providers.md) to construct trust stores. When referencing a namespaced KMP resource, ensure to include the corresponding namespace prefix, while cluster-wide KMP should be referenced by its name directly. Refer to [this section](../../reference/custom%20resources/key-management-providers.md#utilization-in-verifiers) for more information.
+
+
+> NOTE 2: `verificationCertStores` supersedes `verificationCerts` if both fields are specified.
+
+> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  This change is backward compatible, the implementation supports both original CRs, which contain no trust store type, and CRs specified `trust-store-type` for each certification collection. In the sample above, `ca:certs` is the trust store specified and the `certs` suffix corresponds to the `certs` certification collection listed in the `verificationCertStores` section.
```
Comment (binbin-li) [inline docs/plugins/verifier/notation.md:91]:
```suggestion
> NOTE 3: `verificationCertStores` currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`(coming soon).  This change is backward compatible, the implementation supports both original CRs, which contain no trust store type, and CRs specifying `trust-store-type` for each certificate collection. In the sample above, `ca:certs` is the trust store specified and the `certs` suffix corresponds to the `certs` certification collection listed in the `verificationCertStores` section. 
```

**Review by binbin-li — APPROVED** [review]
one minor comment, lgtm

File: docs/plugins/verifier/notation.md (hunk 1 above)
Comment (junczhu) [reply docs/plugins/verifier/notation.md:91]: resolved

**Review by binbin-li — APPROVED** [review]

**Pull request merged**